### PR TITLE
Official version for the release

### PR DESCRIPTION
**Pull request by simoatze**
@dongahn @jprotze Hi have merged the master changes into "towards_tr4" branch in this way the pull request will merge automatically. Since Joachim created a new stable branch for the OpenMP runtime with OMPT I think we should start merging this into master and get ready for the release.

### CODE REVIEW

**Comment by dongahn** [comment]
Thanks @simoatze! I think @jprotze is the most qualified reviewer for this.

**Comment by jprotze** [comment]
I will need to look closely into the "stable" branch. The first version to choose the HEAD for that branch was a rough shot, but seems to be broken. I'll look into that tonight

File: tools/clang-archer++.in [hunk 1]
```diff
@@ -68,4 +68,4 @@ else
   link_flags=""
 fi
 
-@LLVM_ROOT@/bin/clang++ -I@OMP_PREFIX@/include -Xclang -load -Xclang @CMAKE_INSTALL_PREFIX@/lib/LLVMArcher.so -fopenmp -fsanitize=thread $link_flags -g "$@"
+@LLVM_ROOT@/bin/clang++ -I@OMP_PREFIX@/include -Xclang -load -Xclang @CMAKE_INSTALL_PREFIX@/lib/LLVMArcher.so -std=c++11 -fopenmp -fsanitize=thread $link_flags -g "$@"
```
Comment (dongahn) [inline tools/clang-archer++.in]: What is the main reason should Archer only accept the code conforming to C++11? Should Archer fail to compile if the users' code use some non-standard features only available in Clang or GCC, which stray away form C++11? 

Would the old way be better -- we don't specify the C++ standard in this wrapper, which still allow the user to specify a standard if they want?

Comment (simoatze) [inline tools/clang-archer++.in]: For every c++ program that I am compiling I am getting compiler errors, and I always have to add "-std=c++11", but yeah I can't generalize, I'll revert it.

**Comment by dongahn** [comment]
@simoatze: yeah.. the code written to C++11 should specify -std=c++1 by itself as they would with other compilers.

**Comment by dongahn** [comment]
Is this ready to go in? I don't think I am fully qualified to merge this though so I will need @jprotze's okay.

**Comment by simoatze** [comment]
@dongahn Now that @jprotze  reached a stable release for the OpenMP runtime we are good to go. I double checked everything and looks good. You can either merge or we can wait for a double check from @jprotze.

**Comment by dongahn** [comment]
I'd be honored to merge this in :-)

**Comment by simoatze** [comment]
Wow! Ready to release!

**Comment by simoatze** [comment]
@dongahn The new release will be 1.0.0 right?

**Comment by dongahn** [comment]
Yes v1.0.0

**Pull request merged**